### DISCUSSION TITLE
Number of total plus dead, asleep, and awake nodes in control panel

### DISCUSSION THREAD

**Discussion by jhein05** [zwave-js/zwave-js-ui#1560] [Ideas]
Seeing the total number of nodes and the number of dead, asleep, and awake nodes in the header of the control panel could be a helpful things to see the overall status of your Z-Wave network faster.

**Comment by robertsLando** [comment]
Would you like to see this info in control panel or zwave graph?

**Reply by jhein05** [reply]
Personally, I would think control panel, because it is the first thing you see, which makes it the perfect location for something that is supposed to give you a first impression of the status of your network.

**Reply by robertsLando** [reply]
Opened a feature to track this :) thanks
